docs/en/latest/admin-api.md

```diff
@@ -564,7 +564,7 @@ The following should be considered when setting the `hash_on` value:
 - When set to `vars_combinations`, the `key` is required. The value of the key can be a combination of any of the [Nginx variables](http://nginx.org/en/docs/varindex.html) like `$request_uri$remote_addr`.
 - When no value is set for either `hash_on` or `key`, the key defaults to `remote_addr`.
 
-The features described below requires APISIX to be run on [APISIX-OpenResty](./how-to-build.md#step-6-build-openresty-for-apache-apisix):
+The features described below requires APISIX to be run on [APISIX-Base](./how-to-build.md#step-6-build-openresty-for-apache-apisix):
```

Comment:
There is no `how-to-build.md#step-6-build-openresty-for-apache-apisix` in the latest how-to-build document, we need to change it to another link. We can add step 6 to the FAQ document. Or do we have any other option?

Reply:
Yes, @oil-oil please check https://apisix.apache.org/docs/apisix/next/how-to-build